docs/introduction/Sidechain-SDK-extension.rst

```diff
@@ -422,21 +420,21 @@ For a successful response, implement SuccessResponse interface with data to be r
     }
     }
 
-In such case API response will be represented in the following JSON format:
+In such a case, the API response will be represented in the following JSON format:
```

Comment:
@zennermac 

In this case, the API response will be represented by the following JSON format: